bitnami/logstash/templates/headless-svc.yaml

```
@@ -25,5 +25,8 @@ spec:
   type: ClusterIP
   clusterIP: None
   ports:
-    {{- include "common.tplvalues.render" (dict "value" .Values.service.ports "context" $) | nindent 4 }}
+  {{- range $service := .Values.service.ports }}
+    - port: {{ $service.port }}
+      {{- (omit (omit $service "nodePort") "port") | toYaml | nindent 6 }}
+  {{- end }}
```

```suggestion
  {{- range $service := .Values.service.ports }}
    - name: {{ $service.name }}
      port: {{ $service.port }}
      targetPort: {{ $service.targetPort }}
      protocol: {{ $service.protocol }}
  {{- end }}
```

A bit more verbose but IMO it's also more intuitive and avoids using the nested `omit`.